src/state/_common/selectors.js

```diff
@@ -47,38 +47,65 @@ const getAll = (getSubstate) => {
 	);
 };
 
+function modelsFromIndex(models, index) {
+	if (!index || !index.index) {
+		return null;
+	}
+
+	const indexedModels = [];
+	for (let i = 1; i <= index.count; i++){
+		const modelKey = index.index[i];
+		if (modelKey){
+			const indexedModel = models[modelKey];
+			if (indexedModel){
+				indexedModels.push(indexedModel);
+			} else {
+				indexedModels.push({key: modelKey});
+			}
+		} else {
+			indexedModels.push(null);
+		}
+	}
+
+	return indexedModels.length ? indexedModels : null;
+}
+
+function modelsFromIndex2(models, index) {
```

Comment:
More descriptive naming?

Reply:
lik this:
- `modelsFromIndex` => `modelsFromEntireIndex` (https://github.com/gisat-panther/ptr-state/blob/57f3af0874271e046ac01408cdb036b624de8e3b/src/state/_common/selectors.js#L56)
- `modelsFromIndex2` => `modelsFromLoadedIndex` (https://github.com/gisat-panther/ptr-state/blob/57f3af0874271e046ac01408cdb036b624de8e3b/src/state/_common/selectors.js#L79)
?